posthog/models/cohort.py

```diff
@@ -89,7 +91,68 @@ class Cohort(models.Model):
 
     objects = CohortManager()
 
+    # deprecated
+    groups: models.JSONField = models.JSONField(default=list)
```

Comment:
Yeah, I can add the deprecation in the follow up PR, but it would be deprecated once that goes in so 🤷 .

I don't quite get what's wrong with the current solution? The way it right now works is, both the old write path (groups), and the new write path (filters) populate the same property, `properties`, so everything else doesn't need to know where the data is coming from.

Reply:
Oooo I didn't realize the other PR had the other half of this - all good here. Makes sense to me.